### DISCUSSION TITLE
Welcome to Templater_Templates Discussions!

### DISCUSSION THREAD

**Discussion by chhoumann** [chhoumann/Templater_Templates#1] [Announcements]
## 👋 Welcome!
  Hey! If you have any questions or suggestions, feel free to reach out :)

**Comment by moorebi** [comment]
Just watched the demo on `quickadd` and followed all items (i think) ... but when I insert the templater template the popup dialog box comes up and works great but once I enter "a thought" I get an Template Parsing error... aborting.. any thoughts/suggestions?

**Reply by chhoumann** [reply]
Hey! QuickAdd is now a plugin, and can be found in the community plugins store. You should use that instead.

**Reply by moorebi** [reply]
> Hey! QuickAdd is now a plugin, and can be found in the community plugins store. You should use that instead.

Yeap just saw that thanks...
